### PR TITLE
Aj user posts

### LINKED ISSUE #24: View User Profile Details
As an admin, I would like to see the content of a User Profile so I can read it.

**Given** a user is viewing a list of User Profiles  
**When** they select a user
**Then** they should be directed to a User Profile Detail page

User Profile Details include:

* Full name
* Avatar image (if exists, else use a default image)
* Display name
* Email
* Creation Date (MM/DD/YYYY)
* User Profile type

### PR DESCRIPTION
**Pull request by Adanieljohnson91**
Details:
Added User Details page. 
Fixes: #24 
Testing Instructions:
Go to userprofiles page and click the users name,
this will take you to the user details page.
Users information should be displayed 
[X] New Feature
[X] Comments added to hard to understand code.

### CODE REVIEW

**Review by johnhester — APPROVED** [review]
The user profiles look great!

**Review by dLars99 — APPROVED** [review]
Good to go after resolving merge conflict

**Pull request closed without merging**
